salt/modules/state.py

```diff
@@ -285,7 +293,6 @@ def request(mods=None,
         salt '*' state.request test
         salt '*' state.request test,pkgs
     '''
-    kwargs['test'] = True
```

Comment:
OK. I will have to ask him about it. I suspect this is a regression but he also wrote it originally so I'm a bit confused. Thanks for the reply.